### ISSUE TITLE
support PySide6

### ISSUE THREAD

**Issue by chowooseung** [nxt-dev/nxt_editor#278]
**Describe the solution you'd like**
Are there plans to support PySide6 because Maya2025 has switched to it?

**Comment by ImLucasBrown** [comment]
Yes, I've been working on it. We've dropped Py 2 support as we move towards PySide6. I can't promise a timeline, this is still mostly a side project for us. 🙂

**Comment by satishgoda** [comment]
I made some changes to get the UI up and running, but there are still a LOT of things to convert.

**Comment by ImLucasBrown** [comment]
Here's the branch I've been working on. Most of the changes needed to get the editor open and doing the basics are in there: https://github.com/nxt-dev/nxt_editor/tree/features/qt/pyside_6

**Comment by satishgoda** [comment]
Thanks for sharing. I will pull your branch and check to see if I have something to contribute from my changes.

**Comment by ImLucasBrown** [comment]
[dev](https://github.com/nxt-dev/nxt_editor/tree/dev) now supports PySide6. Still working out a few visual issues, but it's functional. You'll also need to use the latest [dev branch of the core](https://github.com/nxt-dev/nxt/tree/dev) if you're trying to run in Python 3.11.
